db/migrate/20220812142820_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :post, null: false, foreign_key: true
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] As per the given ERD diagram _user_ on the _comment_ model is referenced as an _author_, similar to:
```ruby
t.references :author, foreign_key: { to_table: 'users' }
```

Reply:
Hi @rloterh, I have updated according to the ERD as you suggested. Thanks for the feedback.